### PR TITLE
Improvements to Models section

### PR DESCRIPTION
**Pull request by lukekim**

### CODE REVIEW

File: spiceaidocs/docs/components/models/anthropic.md [hunk 1]
```diff
@@ -1,36 +1,30 @@
...
 sidebar_label: 'Anthropic'
 sidebar_position: 5
 ---
 
-To use a language model hosted on Anthropic, specify `anthropic` in `from`.
+To use a language model hosted on Anthropic, specify `anthropic` in the `from` field.
 
-For a specific model, include it as the model ID in `from` (see example below). Defaults to `"claude-3-5-sonnet-20240620"`.
-These parameters are specific to Anthropic models:
+To use a specific model, include its model ID in the `from` field (see example below). If not specified, the default model is `"claude-3-5-sonnet-20240620"`.
```
Comment (Jeadie) [inline spiceaidocs/docs/components/models/anthropic.md]: Default is now `"claude-3-5-sonnet-latest"`
 https://github.com/spiceai/spiceai/blob/cedefdb5b9e6860cdb3cd71303454f207fb122a7/crates/llms/src/anthropic/mod.rs#L38

File: spiceaidocs/docs/components/models/filesystem.md [hunk 2]
```diff
@@ -1,17 +1,42 @@
 ---
 title: 'Filesystem'
+description: 'Instructions for using machine learning models hosted on a filesystem with Spice.'
```
Comment (Jeadie) [inline spiceaidocs/docs/components/models/filesystem.md]: > machine learning models hosted

Might confuse users who don't recognize LLMs are ML.

File: spiceaidocs/docs/components/models/filesystem.md [hunk 3]
```diff
@@ -1,17 +1,42 @@
...
     name: local_fs_model
-    datasets:
-      - taxi_trips
+```
+
+### Example: Loading a GGUF Model
+
+```yaml
+models:
+  - from: file://absolute/path/to/my/model.ggml
```
Comment (Jeadie) [inline spiceaidocs/docs/components/models/filesystem.md]:
```suggestion
  - from: file://absolute/path/to/my/model.gguf
```

File: spiceaidocs/docs/components/models/filesystem.md [hunk 4]
```diff
@@ -1,17 +1,42 @@
...
+    name: local_ggml_model
+```
+
+### Example: Loading a SafeTensor Model
+
+```yaml
+models:
```
Comment (Jeadie) [inline spiceaidocs/docs/components/models/filesystem.md:43]: safe tensors need lots of additional files. Here's a complete example
```yaml
models:
  - name: safety
    from: file:models/llms/llama3.2-1b-instruct/model.safetensors
    params:
      model_type: llama3
    files:
      - path: models/llms/llama3.2-1b-instruct/tokenizer.json
      - path: models/llms/llama3.2-1b-instruct/tokenizer_config.json
      - path: models/llms/llama3.2-1b-instruct/config.json
```

Comment (Jeadie) [reply spiceaidocs/docs/components/models/filesystem.md:43]: Same for GGML.

File: spiceaidocs/docs/components/models/anthropic.md (hunk 1 above)
Comment (lukekim) [inline spiceaidocs/docs/components/models/anthropic.md]:
```suggestion
To use a specific model, include its model ID in the `from` field (see example below). If not specified, the default model is `"claude-3-5-sonnet-latest"`.
```

File: spiceaidocs/docs/components/models/filesystem.md (hunk 2 above)
Comment (lukekim) [inline spiceaidocs/docs/components/models/filesystem.md]:
```suggestion
description: 'Instructions for using models hosted on a filesystem with Spice.'
```

File: spiceaidocs/docs/components/models/anthropic.md [hunk 5]
```diff
@@ -1,36 +1,30 @@
...
+| `anthropic_api_key`    | The Anthropic API key.           | -                              |
+| `anthropic_auth_token` | The Anthropic auth token.        | -                              |
+| `endpoint`             | The Anthropic API base endpoint. | `https://api.anthropic.com/v1` |
+
+Example `spicepod.yml` configuration:
 
 ```yaml
 models:
-  - from: anthropic:claude-3-5-sonnet-20240620
+  - from: anthropic:claude-3-5-sonnet
```
Comment (Jeadie) [inline spiceaidocs/docs/components/models/anthropic.md]:
```suggestion
  - from: anthropic:claude-3-5-sonnet-latest
```

**Review by Jeadie — APPROVED** [review]

**Pull request merged**